mapping.go

```diff
@@ -45,6 +45,16 @@ func NewTextFieldMapping() *mapping.FieldMapping {
 	return mapping.NewTextFieldMapping()
 }
 
+// NewKeywordFieldMapping returns a default field mapping for structured
+// content which are typically used for filtering, sorting, aggregations.
+// Keyword fields are only searchable by their exact value.
+// For example, IDs, email addresses, hostnames, tags, etc.
+// Avoid using keyword fields for full-text search (use the `text` field type
+// instead).
```

Comment:
Sorry, but I don't like this verbiage either.
Prefer to keep it simple as ..

    NewKeywordFieldMapping returns a field mapping for text using the keyword
    analyzer, which essentially doesn't apply any specific text analysis.